client/src/app.jsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+  import React, { Component } from 'react';
+  import axios from 'axios';
 
-const App = () => (
-  <div>
-    App me down!
-  </div>
-);
+  export default class App extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        listingId: null
+      };
+      this.houseData = {};
+    };
 
-export default App;
+    componentDidMount() {
+      if(this.props.listingId) {
+        axios.get(`/house?listingId=${this.props.listingId}`)
+          .then((response) => {
+            this.houseData = response.data[0];
+            this.setState({
+              listingId: this.houseData.listingId
+            });
+          });
+      };
+    };
+
+    render() {
+      return (
+        <div>
+          Mamba4Life
+          <p>{
+            this.state.listingId === null ? 'Hello' : `${this.houseData.name}`}
```

Comment:
You don't need the `${}` in JSX you could just write {}.  Both work so it is more stylistic, but it can save you two characters of typing.